packer/conf/hooks/environment

```diff
@@ -19,11 +14,21 @@ if [[ -n "${BUILDKITE_SECRETS_BUCKET:-}" ]] && [[ -z "${BUILDKITE_SECRETS_KEY:-}
   exit 1
 fi
 
-SSH_KEY_NAME="${SSH_KEY_NAME:-id_rsa_github}"
-
-# startup an ssh-agent with the ssh-key
 eval "$(ssh-agent -s)"
-ssh-add <(download_github_key "$SSH_KEY_NAME")
+
+if [[ -n "${BUILDKITE_SECRETS_BUCKET:-}" ]] ; then
+  [[ -d ~/secrets ]] || mkdir -p ~/secrets
+
+  echo "Downloading ssh key from $BUILDKITE_SECRETS_BUCKET"
+  if s3_download "${SSH_KEY_NAME:-id_rsa_github}" ~/secrets/id_rsa_github ; then
+    ssh-add ~/secrets/id_rsa_github
```

Comment:
What was the reason to move away from <()?

It seems a little nicer then having to manually clean up these files.